index.html

```diff
@@ -546,10 +547,11 @@ <h3>Integrity Validation</h3>
         <section>
           <h3>Evaluation</h3>
           <p>
-            Validation of a given Credential against its schema is to be performed according
-            to its associated [[JSON-SCHEMA]] specification. If validation
-            succeeds the credential is considered to be valid against its
-            <a>credential schema</a>.
+            Validation of a given credential against its schema is to be performed according
+            to its associated [[JSON-SCHEMA]] specification. If validation succeeds the credential
+            is considered to be valid against its <a>credential schema</a>. In the case that 
+            an implementer encounters a schema whose version it does not support, processing
+            cannot continue, and validation MUST be halted. 
```

Comment:
```suggestion
            to its associated [[JSON-SCHEMA]] specification. Validation MUST fail when the credential
            is considered not valid against its <a>credential schema</a> or if the JSON Schema
            version of the <a>credential schema</a> is not supported by the implementer. 
```

Reply:
I'm not sure I want to say that validation is failed since there's no way to determine the state of validity if the given schema cannot be processed

Reply:
imagine a function that returned <boolean, error> then if you could process the schema you'd always return true/false and no error. an error would be returned if you could not process the schema

Reply:
It seems there should be a third result — valid, indeterminate, not valid.

Reply:
updated to reflect three possible result states